src/SayMoreTests/model/ProjectTests.cs

```diff
@@ -409,9 +423,9 @@ private void WriteXmlResource(string file)
 				if (!Directory.Exists(SessionScenarioPath))
 					Directory.CreateDirectory(SessionScenarioPath);
 				SessionScenarioPath = Path.Combine(SessionScenarioPath, "ContributorsScenario");
-				if (!Directory.Exists(SessionScenarioPath))
-					Directory.CreateDirectory(SessionScenarioPath);
 			}
+			if (!Directory.Exists(SessionScenarioPath))
```

Comment:
Checking for existence (while a common practice) is technically unnecessary since CreateDirectory will do it again.